File: climb_Stairs.py
class Solution(object):
    def climbStairs(self, n):
        """
        :type n: int
        :rtype: int
        """
        if n == 1:
            return 1
        if n == 2:
            return 2

        first, second = 1, 2
        for i in range(3, n + 1):
            total = first + second
            first, second = second, total
        return second

File: easy/climb_Stairs.java
class Solution {
    public int climbStairs(int n) {
        if(n==1)return n;
        if(n==2)return n;
        int a=0;
        int b=1;
        int sum=0;
        for(int i=0;i<n;i++){
            sum=a+b;
            a=b;
            b=sum;

            }
            return sum;
        }
        
    }
